### PR TITLE
docs: new installation method and links

### PR DESCRIPTION
**Pull request by riivx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
-   ```
-   in the new input box labelled **VENDETTA URL**.
-   - If the **Developer** settings section does not appear, restart the app
-5. Restart the app and check if it says Revenge in the settings
+> While Revenge is platform-agnostic, there is no guarantee it will always work on iOS. I do not have a device to test the changes I've made, so feel free to report iOS-specific issues if you encounter one.
 
+1. Install [Vendetta Manager](https://github.com/vendetta-mod/VendettaManager/releases/latest/download/Manager.apk)
+2. Download (don't install!) [latest Xposed module](https://github.com/revenge-mod/RevengeXposed/releases/latest/download/app-release.apk)
+3. Go to Vendetta manager's **Settings > About** and press version number **10 times**
+4. Set the custom Xposed module location to *`/storage/emulated/0/Download/app-release.apk`*
```
Comment (PalmDevs) [inline README.md:15]: Bold and also capitalize the setting name here:

```
Set the **Custom Xposed module location** [...]
```

Also use a codeblock for the path so it's easily copy-able.

What happens if the user downloads it to somewhere else other than that path? should probably include some text above saying this is the default path?

File: README.md [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
-   ```
-   https://cdn.jsdelivr.net/gh/revenge-mod/builds@main/revenge.js
-   ```
-   in the new input box labelled **VENDETTA URL**.
-   - If the **Developer** settings section does not appear, restart the app
-5. Restart the app and check if it says Revenge in the settings
+> While Revenge is platform-agnostic, there is no guarantee it will always work on iOS. I do not have a device to test the changes I've made, so feel free to report iOS-specific issues if you encounter one.
 
+1. Install [Vendetta Manager](https://github.com/vendetta-mod/VendettaManager/releases/latest/download/Manager.apk)
+2. Download (don't install!) [latest Xposed module](https://github.com/revenge-mod/RevengeXposed/releases/latest/download/app-release.apk)
```
Comment (PalmDevs) [inline README.md:13]:
```diff
- latest
+ the latest
```

File: README.md [hunk 3]
```diff
@@ -1,31 +1,28 @@
...
+3. Go to Vendetta manager's **Settings > About** and press version number **10 times**
+4. Set the custom Xposed module location to *`/storage/emulated/0/Download/app-release.apk`*
+5. Install!
```
Comment (PalmDevs) [inline README.md:16]: should probably say to head back to the first screen first.

**Review by PalmDevs — CHANGES_REQUESTED** [review]
A few changes and we're good 👍🏻

**Comment by riivx** [comment]
done!

**Review by PalmDevs — APPROVED** [review]
LGTM

**Pull request merged**